template/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "react": "16.8.1",
```

Comment:
React version was bumped to 16.9.0

Reply:
That's what they wrote in the changelog, but they actually haven't done so: https://github.com/facebook/react-native/blob/v0.61.1/template/package.json

FYI: I don't add the changes hand by hand, I use the template folder from the official repo.

Reply:
You can also check the upgrade helper and there you'll see as well that the React version hasn't been changed: https://react-native-community.github.io/upgrade-helper/?from=0.60.6&to=0.61.1